src/aliceVision/sfmData/colorize.cpp

```diff
@@ -91,7 +92,7 @@ void colorizeTracks(SfMData& sfmData)
                 uv.y() = clamp(uv.y(), 0.0, static_cast<double>(image.Height() - 1));
                 const image::RGBColor obsColor = image(uv.y(), uv.x());
                 image::RGBfColor rgbf(obsColor.r(), obsColor.g(), obsColor.b());
-                #pragma omp critical
+                #pragma omp critical (i)
```

Comment:
It appears impossible:
https://stackoverflow.com/questions/62473543/openmp-named-critical-section-if-a-program-variable-is-used-is-it-evaluated-or

"i" is just used as a label here.